### PR TITLE
DOC-7888 Collection Manager

### PR DESCRIPTION
**Pull request by osfameron**

### CODE REVIEW

File: modules/howtos/examples/collection-manager.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    try {
+        $collections->createScope("example-scope");
+    }
+    catch (Exception $e) {
+        print $e;
+    }
+    // end::create-scope[]
+
+    print "scopeAdmin\n";
+	// tag::scopeAdmin[]
```
Comment (maria-robobug) [inline modules/howtos/examples/collection-manager.php]: got an extra space here I think?

File: modules/howtos/examples/collection-manager.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    $user->setUsername("scopeAdmin");
+    $user->setDisplayName("Manage Collections in Scope [travel-sample:*]");
+    $user->setPassword("password");
+    $user->setRoles([
+        (new \Couchbase\Role)->setName("scope_admin")->setBucket("travel-sample")->setScope("example-scope"),
+        (new \Couchbase\Role)->setName("data_reader")->setBucket("travel-sample")
+    ]);
+
+    $users->upsertUser($user);
+	// end::scopeAdmin[]
```
Comment (maria-robobug) [inline modules/howtos/examples/collection-manager.php]: same here, extra space

File: modules/howtos/examples/collection-manager.php [hunk 3]
```diff
@@ -0,0 +1,101 @@
+<?php
+
+use \Couchbase\ClusterOptions;
+use \Couchbase\Cluster;
+use \Couchbase\User;
+
+function getCollections($username, $password) {
+    print "create-collection-manager\n";
+
+	// tag::create-collection-manager[]
```
Comment (maria-robobug) [inline modules/howtos/examples/collection-manager.php]: extra space here

File: modules/howtos/examples/collection-manager.php [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    $options = new \Couchbase\ClusterOptions();
+    $options->credentials($username, $password);
+    $cluster = new \Couchbase\Cluster("localhost", $options);
+    $bucket = $cluster->bucket("travel-sample");
+
+    $collections = $bucket->collections();
+	// end::create-collection-manager[]
```
Comment (maria-robobug) [inline modules/howtos/examples/collection-manager.php]: same here

File: modules/howtos/examples/collection-manager.php [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+use \Couchbase\ClusterOptions;
+use \Couchbase\Cluster;
+use \Couchbase\User;
+
+function getCollections($username, $password) {
+    print "create-collection-manager\n";
+
+	// tag::create-collection-manager[]
+    $options = new \Couchbase\ClusterOptions();
```
Comment (maria-robobug) [inline modules/howtos/examples/collection-manager.php:7]: Do we need to re-use the fully qualified name here if we've actually imported above with `use`?
I'm never sure about this with PHP 😕

**Review by maria-robobug — APPROVED** [review]
LGTM, just had one question about the imports 👍

File: modules/howtos/examples/collection-manager.php (hunk 1 above)
Comment (osfameron) [reply modules/howtos/examples/collection-manager.php]: woo, so what's happened is that Go uses tabs, and I copied the examples from the Go project while translating them. So any thing I typed new was with spaces, but the tags were left unchanged, so stayed as tabs...

Didn't show up on my editor of course... will fix now, thanks.

File: modules/howtos/examples/collection-manager.php (hunk 5 above)
Comment (osfameron) [reply modules/howtos/examples/collection-manager.php:7]: I think can omit, but may be clearer to keep it, and remove the explicit import (which I copied from a previous example...)

**Comment by osfameron** [comment]
Merging as review issues addressed with no further comments.

**Pull request merged**